python/sglang/srt/model_executor/model_runner.py

```diff
@@ -153,6 +153,13 @@ def __init__(
         min_per_gpu_memory = self.init_torch_distributed()
         self.sampler = Sampler()
         self.load_model()
+        if self.tp_size > 1:
+            logger.info(f"Tensor parallelism is enabled, {self.tp_size} devices will be used.")
```

Comment:
Will this break other models? Can we only do this for torch_native_llama?

Reply:
Good catch. Added a `supports_torch_tp` attr in model.